<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstallationPackageClass</class>
 <widget class="QMainWindow" name="InstallationPackageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Adobe Devanagari</family>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>InstallationPackage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: &quot;微软雅黑&quot; ;</string>
  </property>
  <property name="windowFilePath">
   <string>font:</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_ChangePath">
    <property name="geometry">
     <rect>
      <x>603</x>
      <y>450</y>
      <width>78</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(57, 59, 64);
color:rgb(244, 216, 168);
border:none;
border-radius:4px;
}
QPushButton:hover {
background-color:rgb(77, 79, 83);
}
QPushButton:pressed {
background-color:rgb(57, 59, 64);
}</string>
    </property>
    <property name="text">
     <string>更改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Path">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>450</y>
      <width>568</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-radius:2px;
border:1px solid rgb(236, 236, 236);
background-color:rgb(255, 255, 255);

color:rgb(125, 125, 125);
font: 10pt &quot;微软雅黑&quot; ;
}</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>C:\Program Files\Genshin Impact</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Custom">
    <property name="geometry">
     <rect>
      <x>611</x>
      <y>365</y>
      <width>69</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(255, 255, 255, 0);
border:none;
font: &quot;微软雅黑&quot; ;
color:rgb(255, 255, 255);
font-size:14px;
}</string>
    </property>
    <property name="text">
     <string>自定义安装</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ShowLisence">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>362</y>
      <width>103</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgba(255, 255, 255, 0);
border:none;
color:rgb(250, 220, 175);
font-size:14px;
}</string>
    </property>
    <property name="text">
     <string>《软件免责协议》</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Bg_TopRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>710</width>
      <height>400</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Myungjo Std M</family>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(:/InstallationPackage/Resource/bg.jpg)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Bg_BottomRect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>710</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Titile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>710</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Title_Mini">
    <property name="geometry">
     <rect>
      <x>645</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin0.png)
}
QPushButton:hover {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}
QPushButton:pressed {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Title_Exit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(0, 0, 0);
border:none;
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin0.png)
}
QPushButton:hover {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}
QPushButton:pressed {
background-image:url(:/TitleButtonImage/resource/titlebutton/bmin1.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_IsAgree">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: &quot;微软雅黑&quot; ;
color:rgb(255, 255, 255);
font-size:14px;</string>
    </property>
    <property name="text">
     <string>已阅读并认可</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Install">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>277</y>
      <width>190</width>
      <height>54</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>-1</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color:rgb(255, 199, 9);
color:rgb(134, 77, 29);
border:none;
border-radius:4px;

font: &quot;微软雅黑&quot; ;
font-size:20px;
}
QPushButton:hover {
background-color:rgb(255, 204, 33);
}
QPushButton:pressed {
background-color:rgb(255, 204, 33);
}</string>
    </property>
    <property name="text">
     <string>立即安装</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>426</y>
      <width>96</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>选择安装路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Tag_Message">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>493</y>
      <width>221</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(125, 125, 125);</string>
    </property>
    <property name="text">
     <string>所需空间：300MB       可用空间：25GB</string>
    </property>
   </widget>
   <zorder>label_Bg_BottomRect</zorder>
   <zorder>label_Bg_TopRect</zorder>
   <zorder>pushButton_ChangePath</zorder>
   <zorder>lineEdit_Path</zorder>
   <zorder>pushButton_Custom</zorder>
   <zorder>pushButton_ShowLisence</zorder>
   <zorder>label_Titile</zorder>
   <zorder>pushButton_Title_Mini</zorder>
   <zorder>pushButton_Title_Exit</zorder>
   <zorder>radioButton_IsAgree</zorder>
   <zorder>pushButton_Install</zorder>
   <zorder>label_Tag</zorder>
   <zorder>label_Tag_Message</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InstallationPackage.qrc"/>
 </resources>
 <connections/>
</ui>
